rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -283,19 +299,116 @@ private void voteToCommitPendingFederation() {
+    }
+   
+    private void registerSignedSvpFundTx() throws Exception {
+        var pegoutsTxs = bridgeStorageProvider.getPegoutsWaitingForConfirmations()
+            .getEntries().stream().toList();
+        assertEquals(1, pegoutsTxs.size());
+        var svpFundTx = new BtcTransaction(
+            BRIDGE_CONSTANTS.getBtcParams(), pegoutsTxs.get(0).getBtcTransaction().bitcoinSerialize());
+       
+        signInputs(svpFundTx, ORIGINAL_FEDERATION_MEMBERS_KEYS.subList(0, 5));
+
+        var pmtWithTransactions = 
+            BridgeSupportTestUtil.createValidPmtForTransactions(
+                List.of(svpFundTx.getHash()), BRIDGE_CONSTANTS.getBtcParams());
+        var btcBlockWithPmtHeight =
+            BRIDGE_CONSTANTS.getBtcHeightWhenPegoutTxIndexActivates() + BRIDGE_CONSTANTS.getPegoutTxIndexGracePeriodInBtcBlocks();
+        var chainHeight = btcBlockWithPmtHeight + BRIDGE_CONSTANTS.getBtc2RskMinimumAcceptableConfirmations();
+        BridgeSupportTestUtil.recreateChainFromPmt(
+            btcBlockStore, chainHeight, pmtWithTransactions, btcBlockWithPmtHeight, BRIDGE_CONSTANTS.getBtcParams());
+        bridgeStorageProvider.save();
+
+        int activeFederationUtxosSizeBeforeRegisteringTx = federationSupport.getActiveFederationBtcUTXOs().size();
+        bridgeSupport.registerBtcTransaction(
+            buildUpdateCollectionsTx(),
+            svpFundTx.bitcoinSerialize(),
+            btcBlockWithPmtHeight,
+            pmtWithTransactions.bitcoinSerialize()
+        );
+        bridgeSupport.save();
+
+        assertEquals(activeFederationUtxosSizeBeforeRegisteringTx + 1, federationSupport.getActiveFederationBtcUTXOs().size());
+        var svpFundTxHashUnsigned = bridgeStorageProvider.getSvpFundTxHashUnsigned();
+        assertFalse(svpFundTxHashUnsigned.isPresent());
+        var svpFundTransactionSigned = bridgeStorageProvider.getSvpFundTxSigned();
+        assertTrue(svpFundTransactionSigned.isPresent());
+        
+        // simulate removal to leave state clean
+        assertTrue(bridgeStorageProvider.getPegoutsWaitingForConfirmations().removeEntry(pegoutsTxs.get(0)));
+    }
+
+    private void callUpdateCollectionsAndAssertSvpSpendTxIsCreated() throws Exception {
+        // Next call to update collections will create svp spend tx
+        bridgeSupport.updateCollections(UPDATE_COLLECTIONS_TX);
+        bridgeSupport.save();
+
+        var svpFundTxHash = bridgeStorageProvider.getSvpFundTxSigned();
+        assertFalse(svpFundTxHash.isPresent());
+        var svpSpendTransactionHashUnsigned = bridgeStorageProvider.getSvpSpendTxHashUnsigned();
+        assertTrue(svpSpendTransactionHashUnsigned.isPresent());
+        var svpSpendTxWaitingForSignaturesOpt = bridgeStorageProvider.getSvpSpendTxWaitingForSignatures();
+        assertTrue(svpSpendTxWaitingForSignaturesOpt.isPresent());
+    }
+
+    private void registerSvpSpendTx() throws Exception {
```

Comment:
```suggestion
    private void addSignaturesToAndRegisterSvpSpendTx() throws Exception {
```

wdyt? i know is nit but I think is good that the method's name to describe all the important things it does